data_steward/tools/deid_runner.sh

```diff
@@ -105,22 +105,20 @@ bq mk --dataset --force --description "${version} sandbox dataset to apply clean
 python "${CLEANER_DIR}/clean_cdr.py" --project_id "${APP_ID}" --dataset_id "${registered_cdr_deid}" --sandbox_dataset_id "${registered_cdr_deid_sandbox}" --data_stage ${data_stage} --mapping_dataset_id "${cdr_id}"  --cope_lookup_dataset_id "${cope_lookup_dataset_id}" --cope_table_name "${cope_table_name}" -s 2>&1 | tee registered_tier_cleaning_log.txt
 
 cdr_deid_base_staging="${registered_cdr_deid}_base_staging"
-cdr_deid_base_staging_sandbox="${registered_cdr_deid}_base_staging_sandbox"
-cdr_deid_base_sandbox="${registered_cdr_deid}_base_sandbox"
+cdr_deid_base_sandbox="${dataset_release_tag}_deid_base_sandbox"
 cdr_deid_base="${registered_cdr_deid}_base"
 cdr_deid_clean_staging="${registered_cdr_deid}_clean_staging"
-cdr_deid_clean_staging_sandbox="${registered_cdr_deid}_clean_staging_sandbox"
-cdr_deid_clean_sandbox="${registered_cdr_deid}_clean_sandbox"
+cdr_deid_clean_sandbox="${dataset_release_tag}_deid_clean_sandbox"
 cdr_deid_clean="${registered_cdr_deid}_clean"
 
 # Copy cdr_metadata table
-python "${TOOLS_DIR}/add_cdr_metadata.py" --component "copy" --project_id ${APP_ID} --target_dataset ${registered_cdr_deid} --source_dataset ${cdr_id}
+python "${TOOLS_DIR}/add_cdr_metadata.py" --component "copy" --project_id "${APP_ID}" --target_dataset "${registered_cdr_deid}" --source_dataset "${cdr_id}"
 
 # create empty de-id_base dataset to apply cleaning rules
-bq mk --dataset --description "Intermediary dataset to apply cleaning rules on ${registered_cdr_deid}" --label "phase:staging" --label "release_tag:${dataset_release_tag}" --label "de_identified:true" ${APP_ID}:${cdr_deid_base_staging}
+bq mk --dataset --description "Intermediary dataset to apply cleaning rules on ${registered_cdr_deid}" --label "phase:staging" --label "release_tag:${dataset_release_tag}" --label "de_identified:true" "${APP_ID}":"${cdr_deid_base_staging}"
 
 # create empty sandbox dataset to apply cleaning rules on staging dataset
-bq mk --dataset --description "Sandbox created for storing records affected by the cleaning rules applied to ${cdr_deid_base_staging}" --label "phase:sandbox" --label "release_tag:${dataset_release_tag}" --label "de_identified:true" "${APP_ID}":"${cdr_deid_base_staging_sandbox}"
+bq mk --dataset --description "Sandbox created for storing records affected by the cleaning rules applied to ${cdr_deid_base_staging}" --label "phase:sandbox" --label "release_tag:${dataset_release_tag}" --label "de_identified:true" "${APP_ID}":"${cdr_deid_base_sandbox}"
```

Comment:
If the dataset already exists, this will raise
```
BigQuery error in mk operation: Dataset 'aou-res-curation-test:lrwb_sandbox_test' already exists.
```

I think this is ok.  the script should continue.

Reply:
this will continue if the dataset already exists.